spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -327,6 +330,18 @@ private void registerSagaBeanDefinitions(EventProcessingConfigurer configurer) {
         return hierarchy;
     }
 
+    private <A> String prototype(Class<A> type) {
```

Comment:
The name "prototype" is misleading, as this returns the name of the bean (which happens to be a prototype bean)